Modix/Handlers/InvitePurgeHandler.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Discord;
+using Discord.WebSocket;
+using Modix.Services.BehaviourConfiguration;
+using Serilog;
+
+namespace Modix.Handlers
+{
+    public class InviteLinkHandler
+    {
+        private readonly IBehaviourConfiguration _botConfiguration;
+
+        private const string InviteLinkPattern = @"(https?:\/\/)?(www\.)?(discord\.(gg|io|me|li)|discordapp\.com\/invite)\/.+[a-z]";
```

Comment:
Any reason not to simplify this by making it `private static readonly Regex InviteLinkPattern = new Regex(/* ... */);` and getting rid of the `GetRegexCheck` method at the end of the file?

Reply:
Good shout